gen/definitions/profile_parcels/service_lan_vpn.yaml

```diff
@@ -0,0 +1,423 @@
+      example: FW
+    - model_name: ipv4Addresses
+      example: 1.2.3.4
+    - model_name: tracking
+      example: true
+  - model_name: serviceRoute
+    tf_name: service_routes
+    attributes:
+    - model_name: ipAddress
+      tf_name: network_address
+      data_path: [prefix]
+      id: true
+      example: 1.2.3.4
+    - model_name: subnetMask
+      data_path: [prefix]
+      example: 0.0.0.0
+    - model_name: service
+      example: SIG
+    - model_name: vpn
+      example: 0
+  - model_name: greRoute
+    tf_name: gre_routes
+    attributes:
+    - model_name: ipAddress
+      tf_name: network_address
+      data_path: [prefix]
+      id: true
+      example: 1.2.3.4
+    - model_name: subnetMask
+      data_path: [prefix]
+      example: 0.0.0.0
+    - model_name: interface
+      example: gre01
+    - model_name: vpn
+      example: 0
+  - model_name: ipsecRoute
+    tf_name: ipsec_routes
+    attributes:
+    - model_name: ipAddress
+      tf_name: network_address
+      data_path: [prefix]
+      id: true
+      example: 1.2.3.4
+    - model_name: subnetMask
+      data_path: [prefix]
+      example: 0.0.0.0
+    - model_name: interface
+      example: ipsec01
+  - model_name: natPool
+    tf_name: nat_pools
+    attributes:
+    - model_name: natPoolName
+      id: true
+      example: 1
+    - model_name: prefixLength
+      example: 3
+    - model_name: rangeStart
+      example: 1.2.3.4
+    - model_name: rangeEnd
+      example: 2.3.4.5
+    - model_name: overload
+      example: true
+    - model_name: direction
+      example: inside
+      # ====== UX2.0 Tracker and Trackerobject Not Supported By Provider ======  
+    - model_name: refId
+      tf_name: tracker_object_id
+      data_path: [trackingObject, trackerId]
+      exclude_test: true
+      example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+      # test_value: 
+  - model_name: natPortForward
+    tf_name: nat_port_forwards
+    attributes:
+    - model_name: natPoolName
+      id: true
+      example: 2
+    - model_name: sourcePort
+      example: 122
+    - model_name: translatePort
+      example: 330
+    - model_name: sourceIp
+      example: 1.2.3.4
+    - model_name: TranslatedSourceIp
+      example: 2.3.4.5
+    - model_name: protocol
+      example: TCP
+  - model_name: staticNat
+    tf_name: static_nats
+    attributes:
+    - model_name: natPoolName
+      id: true
+      example: 3
+    - model_name: sourceIp
+      example: 1.2.3.4
+    - model_name: TranslatedSourceIp
+      example: 2.3.4.5
+    - model_name: staticNatDirection
+      example: inside
+      # ====== UX2.0 Tracker and Trackerobject Not Supported By Provider ======
+    - model_name: refId
+      tf_name: tracker_object_id
+      data_path: [trackingObject, trackerId]
+      exclude_test: true
+      example: 1b270f6d-479b-47e3-ab0b-51bc6811a303
+      # test_value: 
+  - model_name: nat64V4Pool
+    tf_name: nat_64_v4_pool
+    attributes:
+    - model_name: nat64V4PoolName
```

Comment:
Let's maybe remove the `nat64V4Pool` prefix for those elements as it is redundant:
-> `tf_name: name`
-> `tf_name: range_start`
...